doc/tools/make_rst.py

```diff
@@ -1176,18 +1202,16 @@ def make_rst_class(class_def: ClassDef, state: State, dry_run: bool, output_dir:
                     f.write(make_deprecated_experimental(value, state))
 
                     if value.text is not None and value.text.strip() != "":
-                        f.write(f"{format_text_block(value.text.strip(), value, state)}")
+                        f.write(f"{format_text_block(value.text.strip(), value, state)}\n\n")
                     elif value.deprecated is None and value.experimental is None:
                         f.write(".. container:: contribute\n\n\t")
                         f.write(
                             translate(
-                                "There is currently no description for this enum. Please help us by :ref:`contributing one <doc_updating_the_class_reference>`!"
+                                "There is currently no description for this constant. Please help us by :ref:`contributing one <doc_updating_the_class_reference>`!"
```

Comment:
Not a critique. It seemed like calling them "enum" was an oversight.

Reply:
Yes, enumeration elements are called constants in our terminology.